shared/health/src/main/java/io/pravega/shared/health/Status.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.health;
+
+
+import lombok.Getter;
+
+/**
+ * Enumerates the list of potential states that a service or component may be in.
+ */
+public enum Status {
+    /**
+     * Describes a {@link Status} that is considered in a healthy and operational state.
+     */
+    UP(2),
```

Comment:
Do we need all these states? 

How about mirroring the `status()` from `Service`. All we'd need is New/Starting/Running/Terminated.

I don't know how our components would know to say "warning".

Reply:
Most services that perform similar health checking functionality (which I took inspiration from) seemed to have some sort of 'soft up' state, such as a `WARNING`. Certainly can be removed.